test/nodes/test_retriever.py

```diff
@@ -318,34 +322,49 @@ def test_retribert_embedding(document_store, retriever, docs_with_ids):
         assert isclose(embedding[0], expected_value, rel_tol=0.001)
 
 
-def test_openai_embedding_retriever_selection():
-    # OpenAI released (Dec 2022) a unifying embedding model called text-embedding-ada-002
-    # make sure that we can use it with the retriever selection
-    er = EmbeddingRetriever(embedding_model="text-embedding-ada-002", document_store=None)
-    assert er.model_format == "openai"
-    assert er.embedding_encoder.query_encoder_model == "text-embedding-ada-002"
-    assert er.embedding_encoder.doc_encoder_model == "text-embedding-ada-002"
-    assert er.api_base == "https://api.openai.com/v1"
-
-    # but also support old ada and other text-search-<modelname>-*-001 models
-    er = EmbeddingRetriever(embedding_model="ada", document_store=None)
-    assert er.model_format == "openai"
-    assert er.embedding_encoder.query_encoder_model == "text-search-ada-query-001"
-    assert er.embedding_encoder.doc_encoder_model == "text-search-ada-doc-001"
-    assert er.api_base == "https://api.openai.com/v1"
-
-    # but also support old babbage and other text-search-<modelname>-*-001 models
-    er = EmbeddingRetriever(embedding_model="babbage", document_store=None)
-    assert er.model_format == "openai"
-    assert er.embedding_encoder.query_encoder_model == "text-search-babbage-query-001"
-    assert er.embedding_encoder.doc_encoder_model == "text-search-babbage-doc-001"
+@pytest.mark.unit
+def test_openai_embedding_retriever_model_format():
+    # support text-embedding-ada-002
+    assert "openai" == EmbeddingRetriever._infer_model_format(
+        model_name_or_path="text-embedding-ada-002", use_auth_token=None
+    )
+
+    # support old ada and other text-search-<modelname>-*-001 models
+    assert "openai" == EmbeddingRetriever._infer_model_format(model_name_or_path="ada", use_auth_token=None)
+
+    # support old babbage and other text-search-<modelname>-*-001 models
+    assert "openai" == EmbeddingRetriever._infer_model_format(model_name_or_path="babbage", use_auth_token=None)
+
+    # make sure that we can handle potential unreleased models
+    assert "openai" == EmbeddingRetriever._infer_model_format(
+        model_name_or_path="text-embedding-babbage-002", use_auth_token=None
+    )
```

Comment:
To maintain consistency in the codebase, I would suggest the format `assert computed_value == expected_value`.